vimv

```diff
@@ -12,28 +16,38 @@ trap '{ rm -f "${FILENAMES_FILE}" ; }' EXIT
 if [ $# -ne 0 ]; then
     src=( "$@" )
 else
-    IFS=$'\r\n' GLOBIGNORE='*' command eval 'src=($(ls))'
```

Comment:
perhaps this should be in a separate pull request

Reply:
if somebody wants this feel free.
ive already made it to what i want